.vscode/settings.json

```diff
@@ -10,6 +10,5 @@
   "eslint.enable": true,
   "prettier.singleQuote": true,
   "prettier.trailingComma": "es5",
-  "prettier.parser": "flow",
-  "editor.formatOnSave": true
+  "prettier.parser": "flow"
```

Comment:
why is `formatOnSave` removed? I think we should keep this setting.

If you're having an issue with formating you could override this setting with your user or workspace setting.